src/platforms/common/performance/index.mdx

```diff
@@ -67,7 +67,7 @@ Automatic instrumentation for monitoring the performance of your application is
 
 ## Configure the Sample Rate
 
-Sampling for transactions must also be configured before tracing is enabled in your app. Set the sample rate for your transactions by either:
+First, enable both tracing and configure the sampling rate for transactions. Set the sample rate for your transactions by either:
```

Comment:
```suggestion
Before tracing can be enabled, you have to set the sample rate for your transactions by either:
```
Can sample rates be set BEFORE the tracing package is installed? If so, it would make sense to switch the order and move "configure sample rate" above "enable tracing". It seems illogical to tell people to do something and then say, before that can be done, you should do this.

Reply:
essentially setting a sample rate (directly or indirectly) does both at the same time, there is no really first or before

the tracing package is the performance package, generates spans/transactions and propagates traces

but i understand it is confusing, I would stick with "first, do this to do both..."